assets/js/modules/analytics/components/settings/SettingsForm.js

```diff
@@ -38,17 +39,30 @@ import { MODULES_TAGMANAGER } from '../../../tagmanager/datastore/constants';
 import SettingsControls from './SettingsControls';
 import GA4SettingsControls from './GA4SettingsControls';
 import EntityOwnershipChangeNotice from '../../../../components/settings/EntityOwnershipChangeNotice';
+import SettingsNotice, {
+	TYPE_WARNING,
+} from '../../../../components/SettingsNotice';
+import Link from '../../../../components/Link';
 import { isValidAccountID } from '../../util';
 import { CORE_MODULES } from '../../../../googlesitekit/modules/datastore/constants';
+import { CORE_SITE } from '../../../../googlesitekit/datastore/site/constants';
+import { useFeature } from '../../../../hooks/useFeature';
 const { useSelect } = Data;
 
 export default function SettingsForm( {
 	hasAnalyticsAccess,
 	hasAnalytics4Access,
 } ) {
+	const ga4Enabled = useFeature( 'ga4Reporting' );
 	const accountID = useSelect( ( select ) =>
 		select( MODULES_ANALYTICS ).getAccountID()
 	);
+	const documentationURL = useSelect( ( select ) => {
+		return select( CORE_SITE ).getDocumentationLinkURL( 'ga4' );
+	} );
+	const ga4ModuleConnected = useSelect( ( select ) =>
```

Comment:
Another minor naming point, I realise we do use this name in a couple of places already, but the name `isGA4Connected` is more common. I'd suggest using that name here to help cut down on divergence.

```suggestion
	const isGA4Connected = useSelect( ( select ) =>
```

Reply:
Updated.